config/main.py

```diff
@@ -876,14 +1052,42 @@ def ip(ctx):
 
 @ip.command()
 @click.argument("ip_addr", metavar="<ip_addr>", required=True)
+@click.argument('gw1', metavar='<default gateway IP address>', required=False)
```

Comment:
@pavel-shirshov 
pavel-shirshov 11 days ago Contributor 
should we use gw here? Why gw1?

We will change

Reply:
@pavel-shirshov : Addressed your comment. Changed gw1 to gw.